mix.exs

```diff
@@ -25,7 +25,7 @@ defmodule UeberauthGoogle.Mixfile do
 
   defp deps do
     [{:ueberauth, "~> 0.4"},
-     {:oauth2, "0.6.0"},
+     {:oauth2, "~> 0.8.0"},
```

Comment:
I will switch that to `~> 0.8` - this will allow people to use versions from range `>= 0.8.0 and < 1.0.0` (http://elixir-lang.org/docs/stable/elixir/Version.html)